packages/contracts/scripts/deploy/dollar/solidityScripting/01_Diamond.s.sol

```diff
@@ -0,0 +1,528 @@
+        selectorsOfManagerFacet.push(managerFacet.creditNftAddress.selector);
+        selectorsOfManagerFacet.push(
+            managerFacet.curve3PoolTokenAddress.selector
+        );
+        selectorsOfManagerFacet.push(
+            managerFacet.governanceTokenAddress.selector
+        );
+        selectorsOfManagerFacet.push(
+            managerFacet.sushiSwapPoolAddress.selector
+        );
+        selectorsOfManagerFacet.push(
+            managerFacet.creditCalculatorAddress.selector
+        );
+        selectorsOfManagerFacet.push(
+            managerFacet.creditNFTCalculatorAddress.selector
+        );
+        selectorsOfManagerFacet.push(
+            managerFacet.dollarMintCalculatorAddress.selector
+        );
+        selectorsOfManagerFacet.push(
+            managerFacet.excessDollarsDistributor.selector
+        );
+        selectorsOfManagerFacet.push(managerFacet.masterChefAddress.selector);
+        selectorsOfManagerFacet.push(managerFacet.formulasAddress.selector);
+        selectorsOfManagerFacet.push(managerFacet.stakingShareAddress.selector);
+        selectorsOfManagerFacet.push(
+            managerFacet.stableSwapMetaPoolAddress.selector
+        );
+        selectorsOfManagerFacet.push(
+            managerFacet.stakingContractAddress.selector
+        );
+        selectorsOfManagerFacet.push(managerFacet.treasuryAddress.selector);
+
+        // Access Control
+        selectorsOfAccessControlFacet.push(
+            accessControlFacet.grantRole.selector
+        );
+        selectorsOfAccessControlFacet.push(accessControlFacet.hasRole.selector);
+        selectorsOfAccessControlFacet.push(
+            accessControlFacet.renounceRole.selector
+        );
+        selectorsOfAccessControlFacet.push(
+            accessControlFacet.getRoleAdmin.selector
+        );
+        selectorsOfAccessControlFacet.push(
+            accessControlFacet.revokeRole.selector
+        );
+        selectorsOfAccessControlFacet.push(accessControlFacet.pause.selector);
+        selectorsOfAccessControlFacet.push(accessControlFacet.unpause.selector);
+        selectorsOfAccessControlFacet.push(accessControlFacet.paused.selector);
+
+        // TWAP Oracle
+        selectorsOfTWAPOracleDollar3poolFacet.push(
+            twapOracleDollar3PoolFacet.setPool.selector
+        );
+        selectorsOfTWAPOracleDollar3poolFacet.push(
+            twapOracleDollar3PoolFacet.update.selector
+        );
+        selectorsOfTWAPOracleDollar3poolFacet.push(
+            twapOracleDollar3PoolFacet.consult.selector
+        );
+
+        // Collectable Dust
+        selectorsOfCollectableDustFacet.push(
+            collectableDustFacet.addProtocolToken.selector
+        );
+        selectorsOfCollectableDustFacet.push(
+            collectableDustFacet.removeProtocolToken.selector
+        );
+        selectorsOfCollectableDustFacet.push(
+            collectableDustFacet.sendDust.selector
+        );
+        // Chef
```

Comment:
Missing selector for https://github.com/ubiquity/ubiquity-dollar/blob/fix/deploy-script/packages/contracts/src/dollar/facets/ChefFacet.sol#L37

Reply:
good catch I added it !